relay/src/setup.rs

```diff
@@ -69,6 +69,7 @@ pub fn init_metrics(config: &Config) -> Result<()> {
         &addrs[..],
         default_tags,
         config.metrics_buffering(),
+        config.metrics_aggregation(),
```

Comment:
nit: With this number of arguments, it might be nice to pass a `StatsdConfig` object instead. Not a blocker though.

Reply:
Plan is to get rid of the options all together: https://github.com/getsentry/relay/pull/2425#discussion_r1499565181

What do you think?

Reply:
OK!